apps/web/src/app/(private)/settings/_components/WorkspaceApiKeys/index.tsx

```diff
@@ -44,6 +44,7 @@ export default function WorkspaceApiKeys() {
                     </TableCell>
                     <TableCell>
                       <Tooltip
+                        asChild
```

Comment:
isn't that the default behaviour?

Reply:
I don't think. I had to add it. Not sure if default behaviour makes sense. I don't see recommended on radix docs 
https://www.radix-ui.com/primitives/docs/guides/composition#composing-multiple-primitives